Src/WitsmlExplorer.Api/Workers/DeleteRiskWorker.cs

```diff
@@ -22,36 +25,60 @@ public DeleteRiskWorker(IWitsmlClientProvider witsmlClientProvider)
         {
-            var wellUid = job.RiskReference.WellUid;
-            var wellboreUid = job.RiskReference.WellboreUid;
-            var uid = job.RiskReference.Uid;
-            var deleteRequest = RiskQueries.DeleteRiskQuery(wellUid, wellboreUid, uid);
+            Verify(job);
 
-            var result = await witsmlClient.DeleteFromStoreAsync(deleteRequest);
+            var wellUid = job.RiskReferences.WellUid;
+            var wellboreUid = job.RiskReferences.WellboreUid;
+            var riskUids = job.RiskReferences.RiskUids;
+            var queries = RiskQueries.DeleteRiskQuery(wellUid, wellboreUid, riskUids);
+            bool error = false;
+            var successUids = new List<string>();
+            var errorReasons = new List<string>();
+            var errorEnitities = new List<EntityDescription>();
 
-            if (result.IsSuccessful)
+            var results = await Task.WhenAll(queries.Select(async (query) =>
             {
-                Log.Information("{JobType} - Job successful", GetType().Name);
-                var refreshAction = new RefreshWell(witsmlClient.GetServerHostname(), wellUid, RefreshType.Remove);
-                var workerResult = new WorkerResult(witsmlClient.GetServerHostname(), true, $"Deleted well with uid ${wellUid}");
-                return (workerResult, refreshAction);
-            }
-
-            Log.Error("Failed to delete risk. WellUid: {WellUid}", wellUid);
+                var result = await witsmlClient.DeleteFromStoreAsync(query);
+                var risk = query.Risks.First();
+                if (result.IsSuccessful)
+                {
+                    Log.Information("{JobType} - Job successful", GetType().Name);
+                    successUids.Add(risk.Uid);
+                }
+                else
+                {
+                    Log.Error("Failed to delete risk. WellUid: {WellUid}, WellboreUid: {WellboreUid}, Uid: {riskUid}, Reason: {Reason}",
+                    wellUid,
+                    wellboreUid,
+                    query.Risks.First().Uid,
+                    result.Reason);
+                    error = true;
+                    errorReasons.Add(result.Reason);
+                    errorEnitities.Add(new EntityDescription
+                    {
+                        WellName = risk.WellName,
+                        WellboreName = risk.WellboreName,
+                        ObjectName = risk.Name
+                    });
+                }
+                return result;
+            }));
 
-            var query = RiskQueries.QueryById(wellUid, wellboreUid, uid);
-            var queryResult = await witsmlClient.GetFromStoreAsync(query, new OptionsIn(ReturnElements.IdOnly));
-            EntityDescription description = null;
-            var risk = queryResult.Risks.FirstOrDefault();
-            if (risk != null)
+            var refreshAction = new RefreshWellbore(witsmlClient.GetServerHostname(), wellUid, wellboreUid, RefreshType.Update);
```

Comment:
It should be sufficient to use the existing "RefreshRisks" instead of "RefreshWellbore" here